Running `pnpm install` gave me 

```
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

.
└─┬ @sveltejs/eslint-config
  └── ✕ missing peer eslint-plugin-node@">= 11"
Peer dependencies that should be installed:
  eslint-plugin-node@">= 11"  
```

I'm unfamiliar with pnpm but I'm guessing that this dependency is just missing so I added it. `pnpm install eslint-plugin-node`

That fixed the error but `pnpm install` doesn't update SvelteKit (I think this is because of how the version is pinned?). I changed the version to the latest in the package.json (1.0.0-next.391 as of writing), and ran `pnpm install` again, but this then resulted in another peer dependency missing, this time Vite 3.0.0. I added that and now `pnpm install` runs and it should be on the newest version of SvelteKit.

I also fixed the issue where the package `json` was installed instead of `json5`.

--- a/package.json
+++ b/package.json
@@ -42,15 +42,17 @@
   "dependencies": {
     "@iarna/toml": "^2.2.5",
     "cosmiconfig": "^7.0.1",
+    "eslint-plugin-node": "^11.1.0",
     "html-minifier-terser": "^7.0.0-alpha.1",
     "jsdom": "^19.0.0",
-    "json": "^11.0.0",
+    "json5": "^2.2.1",
     "prettier": "^2.5.1",
-    "tiny-glob": "^0.2.9"
+    "tiny-glob": "^0.2.9",
+    "vite": "^3.0.2"
   },
   "devDependencies": {
     "@sveltejs/eslint-config": "github:sveltejs/eslint-config#v5.8.0",
-    "@sveltejs/kit": "^1.0.0-next.288",
+    "@sveltejs/kit": "^1.0.0-next.391",
     "@typescript-eslint/eslint-plugin": "^5.13.0",
     "@typescript-eslint/parser": "^5.13.0",
     "eslint": "^8.10.0",

